src/plots/cartesian/dragbox.js

```diff
@@ -113,12 +113,14 @@ function makeDragBox(gd, plotinfo, x, y, w, h, ns, ew) {
 
     var dragger = makeRectDragger(plotinfo, ns + ew + 'drag', cursor, x, y, w, h);
 
+    var allFixedRanges = !yActive && !xActive;
+
     // still need to make the element if the axes are disabled
     // but nuke its events (except for maindrag which needs them for hover)
     // and stop there
-    if(!yActive && !xActive && !isSelectOrLasso(fullLayout.dragmode)) {
+    if(allFixedRanges && !isMainDrag) {
         dragger.onmousedown = null;
-        dragger.style.pointerEvents = isMainDrag ? 'all' : 'none';
+        dragger.style.pointerEvents = 'none';
```

Comment:
That is called just once per `dragger` element, right? Ie. it does not need to be reset to pointerEvents `'all'`.

Reply:
Good thinking - I had to check, but yes, if you change it later so this needs `pointerEvents`, it will be reinstated by [`dragElement.init`](https://github.com/plotly/plotly.js/blob/master/src/components/dragelement/index.js#L103)